web/source/keymanweb.js

```diff
@@ -897,7 +897,10 @@
           return null;
         }
 
-        keymanweb.inputList.push(Pelem['kmw_ip']); 
+        keymanweb.inputList.push(Pelem['kmw_ip']);
+        
+        console.log("Unexpected state - this element's simulated input DIV should have been removed from the page!");
```

Comment:
Yeah, you are right. I don't think we want to be coping with that - best to treat an element as gone once it is removed from the page.